courses/free-courses-ar.md

```diff
@@ -336,6 +350,8 @@
 * [Clean Code Book - بالعربي](https://www.youtube.com/playlist?list=PLwWuxCLlF_ufTMlXoJlQvImqz9wIfcWsX) - Omar Ahmed
 * [Creational Design Patterns بالعربى](https://www.youtube.com/playlist?list=PLnqAlQ9hFYdewk9UKGBcHLulZNUBpNSKJ) - Mohammed Reda
 * [Declarative Programming](https://www.youtube.com/playlist?list=PLpbZuj8hP-I6F-Zj1Ay8nQ1rMnmFnlK2f) - درة الاكواد لابن حماد
+* [Head First Object-Oriented Analysis and Design&rlm;](https://www.youtube.com/playlist?list=PLWCr3KJclfzGkG7CXBmSdlIcIpjxo4FbF) - Programming Books
+* [Software Design Patterns](https://www.youtube.com/playlist?app=desktop&list=PLrwRNJX9gLs3oQyBoXtYimY7M5aSF0_oC) - محمد يحيى
```

Comment:
```suggestion
* [Software Design Patterns&rlm;](https://www.youtube.com/playlist?app=desktop&list=PLrwRNJX9gLs3oQyBoXtYimY7M5aSF0_oC) - محمد يحيى
```

Reply:
No need to add `&rlm;` here 
the author name in Arabic, so the context will not swap

Reply:
![image](https://user-images.githubusercontent.com/69223584/180773785-e5ec23d0-89ec-4093-a9ac-25eac056c528.png)

Reply:
I didn't know that, could you add it to #6964?

Reply:
#6964 ? You miss the issue?

Reply:
If you mean this #6714, this only for problem

`* [name in english](url) - name in arabic`
That not cause any problem

Reply:
> If you mean this #6714

ops, i confused id


> That not cause any problem

maybe you could add it just for clarification, in an "extra case 2"

Reply:
> maybe you could add it just for clarification, in an "extra case 2"

done